MAUI/LinearGauge/scale.md

```diff
@@ -0,0 +1,362 @@
+---
+layout: post
+title: Axis in .NET MAUI Linear Gauge control | Syncfusion
+description: Learn here all about adding and customizing Axis of Syncfusion .NET MAUI Linear Gauge (SfLinearGauge) control and more.
+platform: maui
+control: SfLinearGauge
+
+documentation: ug
+---
+
+# Axis in .NET MAUI Linear Gauge (SfLinearGauge)
+
+The Linear Gauge axis is a scale where a set of values can be plotted. An axis can be customized by changing the thickness, color and edge styles. Axis elements such as labels and ticks can also be easily customized and you can also inverse the axis.
+
+## Default axis
+
+By default axis will have the [`Minimum`]() axis value as 0 and the [`Maximum`]() axis value as 100. Without any changes the default axis of the Linear Gauge will be displayed as follows. 
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<gauge:SfLinearGauge />
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfLinearGauge gauge = new SfLinearGauge();
+this.Content = gauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Initialize linear gauge for axis](images/getting-started/default_linear_gauge.PNG)
+
+## Customize minimum and maximum axis values
+
+The [`Minimum`]() and [`Maximum`]() properties of a Linear Gauge can be used to customize the axis values. In the below code snippet the axis is customized to have the [`Minimum`]() value of -50 to [`Maximum`]() value of 50. The axis values are displayed by the labels. Customizing these label styles are further explained in next topics.  
```

Comment:
In the following code sample, the axis is customized to have the [`Minimum`]() value of -50 to [`Maximum`]() value of 50.